Frontend/hooks/useFundraisers.ts

```diff
@@ -17,12 +21,24 @@ const useFundraisers = () => {
       setIsLoadingFundraiser(false);
     };
 
+    const fetchAllFundraiserDonations = async () => {
```

Comment:
I think it's better to create a separate hook, i.e. `useProfile` 
 or `useAccount` to get the data depending on the user account. That hook is only for getting the list of available fundraisers and it's using only for the home page.

Reply:
Sure, Let me work on that right away.